src/app/item-card/item-card.component.scss

```diff
@@ -0,0 +1,51 @@
+.item {
+    max-width: 224px;
+    max-height: 264px;
+    .__container {
+        width: 216px;
+        height: 464px;
+        display: flex;
+        flex-direction: column;
```

Comment:
Why did you decide not to use bootstrap classes for this kind of styles as we agreed to do?

Reply:
Okay!
But for width and height also keep in mind who should be managing the size, if the parent or the card itself.
Having it handled by the parent will give you a more flexible and adaptable component!